File: Ejercicio16.java
import java.util.Scanner;

public class Ejercicio16 {
    public static void main(String[] args){
        System.out.println("ingrese una palabra: ");
        Scanner sc = new Scanner(System.in);
        String s = sc.nextLine();
        int contarA=0;

        for (int i=0; i < s.length(); i++) {
            if(s.charAt(i) == 'a') {
                contarA++;
                System.out.println("van:"+contarA);
            }
        }
        System.out.println("Son "+ contarA+" letras a.");
    }
    
}

File: Ejercicio16.cpp
#include <string>
#include <iostream>
#include <math.h>

using namespace std;

int main(){
    cout << "Ingrese una palabra: " << endl;
    string s;
    getline(std::cin, s);
    int contarA=0;

    for(int i=0; i < s.length(); i++){
        if(s[i] == 'a'){
            contarA++;
        }
    }
    cout << "las letras a son: " <<contarA << endl;
    return 0;
}
